### ISSUE TITLE
Caractère missing in "Réunion" phone mash

### ISSUE THREAD

**Issue by seven-amid** [justdvl/material-ui-phone-number-2#91]
Actually we discover that there is a caractère missing in "re" ( Réunion ) mask phone number. 
here is the correcte format of "re" mask "+262 x xx xx xx xx" but actually we have instead "+.. . .. .. .. .."ru 

check the capture bellow.
<img width="385" alt="ru-mask" src="https://user-images.githubusercontent.com/64092663/216180005-a0a65b86-27d7-40f0-9084-64cd577ca82b.png">

**Comment by igorschlum** [comment]
I agree. it should display phone number as +262 ... and phone numbers are displayed as +26 2 ...
